ccdproc/core.py

```diff
@@ -1740,6 +1741,94 @@ def _sigma_mask(baseline, one_sigma_value, lower_sigma, upper_sigma):
     return mask
 
 
+def bitmask2mask(bitmask, ignore_bits=0, flip_bits=None):
```

Comment:
The thing is that ``ignore_bits`` argument is actually a mask and it is not the thing that gets  converted to a "boolean mask". It is the input DQ array whose elements are "collections of bit flags" (=[bit fields](https://en.wikipedia.org/wiki/Bit_field)).